stdlib/sql/to_privates_test.go

```diff
@@ -117,6 +115,11 @@ func TestSqliteTranslation(t *testing.T) {
 		"DATETIME": flux.TTime,
 	}
 	columnLabel := "apples"
+	quote, err := getQuoteIdentFunc("sqlite3")
+	if !cmp.Equal(nil, err) {
+		t.Log(cmp.Diff(nil, err))
+		t.Fail()
+	}
```

Comment:
It would be totally acceptable (and I think easier to read) to do this here instead:
```Go
if err != nil {
  t.Fail(err)
}
```